### PR TITLE
Issue #211 - add swagger docs

### PR DESCRIPTION
**Pull request by sfradkin**
Added interactive swagger documentation at the `/api` path on the server.

### CODE REVIEW

File: server/app.es6 [hunk 1]
```diff
@@ -34,6 +34,8 @@ app.options('*', function(req, res) {
   res.send();
 });
 
+app.use('/api', express.static('api'));
```
Comment (dubharmonic) [inline server/app.es6]: Can we use a path more descriptive, such as `docs` with the swagger documentation in a swagger directory inside of that?

File: circle.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ machine:
 dependencies:
   pre:
     - yarn global add typedoc
+    # chrome
```
Comment (dubharmonic) [inline circle.yml]: When / how do the swagger docs get generated?

Comment (sfradkin) [inline circle.yml]: I'm using the Swagger UI which generates the interactive docs on the fly from a yml file.  https://swagger.io/swagger-ui/

File: server/app.es6 (hunk 1 above)
Comment (sfradkin) [inline server/app.es6]: Changed URI and location to` docs/api`

**Pull request merged**